src/GoodsList.tsx

```diff
@@ -8,7 +9,15 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
+        key={good.id}
+        data-cy="good"
```

Comment:
Instead of hardcoding the color classes ('is-red', 'is-green', 'is-blue'), consider using dynamic class name based on the color of the good. This will make your code more scalable when there are more colors available.